appendix.rst

```diff
@@ -852,28 +858,28 @@ The Include keyword is optional. It allows you to install additional
 packages into the core operating system. It is a best practice to supply
 only the bare essentials such that the ``%post`` section has what it
 needs to properly complete the build. One common package you may want to
-install when using the ``yum`` build module is YUM or DNF itself.
+install when using the ``yum`` or ``dnf`` build module is YUM or DNF itself.
 
 Notes
 -----
 
-There is a major limitation with using YUM to bootstrap a container. The
+There is a major limitation with using YUM/DNF to bootstrap a container. The
 RPM database that exists within the container will be created using the
 RPM library and Berkeley DB implementation that exists on the host
 system. If the RPM implementation inside the container is not compatible
-with the RPM database that was used to create the container, RPM and YUM
+with the RPM database that was used to create the container, RPM and YUM/DNF
 commands inside the container may fail. This issue can be easily
 demonstrated by bootstrapping an older RHEL compatible image by a newer
 one (e.g. bootstrap a RHEL 8 container from a RHEL 9 host).
 
-In order to use the ``yum`` build module, you must have ``yum``
-installed on your system. It may seem counter-intuitive to install YUM
-on a system that uses a different package manager, but you can do so.
+In order to use the ``yum`` or ``dnf`` build module, you must have ``yum``
+or ``dnf`` installed on your system. It may seem counter-intuitive to install YUM
+or DNF on a system that uses a different package manager, but you can do so.
 For instance, on Ubuntu you can install it like so:
 
 .. code::
 
-   $ sudo apt-get update && sudo apt-get install yum
+   $ sudo apt-get update && sudo apt-get install yum dnf
```

Comment:
No need to list both because they should be the same package.  Pick one.

Reply:
On debian 12 (bookworm) only dnf works, not yum.

Reply:
thanks! removed `yum` only keeping `dnf`